packages/shared/lib/core/network/interfaces/node-info-response.interface.ts

```diff
@@ -1,6 +1,9 @@
 import { IStardustNodeInfo } from './node-info.interface'
 
 export interface INodeInfoResponse {
-    nodeinfo: IStardustNodeInfo
-    url: string
+    payload: {
+        nodeinfo: IStardustNodeInfo
+        url: string
+    }
+    type: 'NodeInfo'
 }
```

Comment:
Don't need to change this as it is changed to the source version on the latest wallet bindings. We are also using a npm package as the dependency now so you won't need to have a local version. And itll stop us all being out of sync 😄